### ISSUE TITLE
Use XFile instead of dynamic to represent file info

### ISSUE THREAD

**Issue by ahmednfwela** [deakjahn/flutter_dropzone#36]
instead of using `dynamic` to represent file info, why not use the existing package [cross_file](https://pub.dev/packages/cross_file) package, which has an `XFile` type that works on all platforms

**Comment by deakjahn** [comment]
Well, the idea seems tempting at first but then, think about it, we don't need to work it on all platforms. This is a specifically web-only package, after all...

**Comment by ahmednfwela** [comment]
yes, but in the future it will need to work on desktop as well, also other packages are adapting it instead of manual implementations https://pub.dev/packages?q=dependency%3Across_file
